Apex7000_BillValidator/PyramidNETRS232.csproj

```diff
@@ -36,11 +36,13 @@
     <DocumentationFile>bin\Release\PyramidNETRS232.xml</DocumentationFile>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="log4net, Version=1.2.15.0, Culture=neutral, PublicKeyToken=669e0ddf0bb1aa2a, processorArchitecture=MSIL">
-      <HintPath>..\packages\log4net.2.0.5\lib\net35-full\log4net.dll</HintPath>
+    <Reference Include="log4net, Version=2.0.9.0, Culture=neutral, PublicKeyToken=669e0ddf0bb1aa2a">
+      <HintPath>..\packages\log4net.2.0.10\lib\net35\log4net.dll</HintPath>
       <Private>True</Private>
     </Reference>
     <Reference Include="System" />
+    <Reference Include="System.Configuration" />
```

Comment:
I must have missed this in the last PR. Are these a log4net dep now?

Reply:
I think this snuck in somehow when I was testing the log4net bump. Can confirm it is not a dependency for log4net 2.0.10 using NET3.5 https://www.nuget.org/packages/log4net/2.0.10

Reply:
Let's remove the System.Web then too if we can.